tests/tests_pytorch/models/test_gpu.py

```diff
@@ -83,8 +83,9 @@ def test_single_gpu_model(tmpdir, devices):
         "-1",
     ],
 )
+@mock.patch.object(CUDAAccelerator, "is_available", return_value=False)
 @mock.patch("lightning_lite.accelerators.mps.MPSAccelerator.is_available", return_value=False)
-def test_root_gpu_property_0_raising(_, devices):
+def test_root_gpu_property_0_raising(n_avail, m_avail, devices):
```

Comment:
```suggestion
def test_root_gpu_property_0_raising(mps_count_0, cuda_count_0, devices):
```

Reply:
Didn't notice the availability of those fixtures, good suggestion. GitHub is indicating "Applying suggestions on deleted lines is currently not supported." so I've gone ahead and pushed the relevant commit rather than applying the suggestion.